### PR TITLE
FHAC-1390: changes for ticket FHAC-1390

### PR DESCRIPTION
**Pull request by balmeeki**
made changes for ticket FHAC-1390

### CODE REVIEW

**Review by jsmith2 — APPROVED** [review]

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/logs.xhtml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+C:\CONNECT\Product\Production\Adapters\General\CONNECTAdminGUI\src\main\webapp\logs.xhtml
+
```
Comment (mhpnguyen) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/logs.xhtml:2]: Verify if this dummy file can be removed?

**Review by mhpnguyen — CHANGES_REQUESTED** [review]
Based on the content, logs.xhtml looks like dummy test page.  Verify the page can be removed

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/logs.xhtml (hunk 1 above)
Comment (jsmith2) [reply Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/logs.xhtml:2]: Out of scope for this ticket.  Agree this file should be removed but let's create a separate ticket to confirm.

**Review by mhpnguyen — APPROVED** [review]

**Review by mhpnguyen — CHANGES_REQUESTED** [review]
Missing https://github.com/CONNECT-Solution/CONNECT/blob/CONNECT_integration/Product/Production/Gateway/CONNECTGatewayWeb/src/main/webapp/index.jsp#L8

**Review by mhpnguyen — APPROVED** [review]

**Pull request merged**